management-packages/standalone-cliplugins/README.md

```diff
@@ -0,0 +1,15 @@
+# standalone-cliplugins Package
+
+This standalone-cliplugins package is currently part of management package plugins to streamline the package release process. This package is not (and should not be) installed on the management cluster. Instead, it is used by the Tanzu CLI directly as an OCI bundle. In future, this package should be placed at more appropriate location.
```

Comment:
also curious, where should this package live?

Reply:
Yes. This might not be the exact place where we want to keep this package in the future and hence I have added the above comment as well as have added more details under the `Note` section in this same file.

However, is there a way we can publish a package and not add it as part of `PackageRepository` ?

Reply:
As discussed in the meeting yesterday, refactoring was done as part of #1061 and this PR has been updated based on that to move this package into standalone packages.